src/main/scala/firrtl/passes/ReplSeqMem.scala

```diff
@@ -88,7 +88,7 @@ Optional Arguments:
     error("No circuit name specified for ReplSeqMem!" + usage)
   )
   val target = CircuitName(passCircuit)
-  def duplicate(n: Named) = this copy (t = (t replace (s"-c:$passCircuit", s"-c:${n.name}")))
+  def duplicate(n: Named) = this copy (t = t replace(s"-c:$passCircuit", s"-c:${n.name}"))
```

Comment:
I have to say I prefer dot notation.  But I was trying to minimize the differences from the previous "un"-stylish code.  I have to admit to relying on IntelliJ to fix lot's of these things in batch.  It would be nice to include our own sensibilities in a style guide but I think that should be  separate or at an extended version of the style-cleanup-take-N passes